lrpar/src/lib/ctbuilder.rs

```diff
@@ -832,7 +837,10 @@ where
             RecoveryKind::CPCTPlus => "CPCTPlus",
             RecoveryKind::None => "None",
         };
-        match self.yacckind.unwrap() {
+        match self.yacckind.clone().unwrap() {
+            YaccKind::SelfDescribing(_) => {
+                unimplemented!("Concrete YaccKind should be known at this point")
```

Comment:
I wonder if this is a "should" or perhaps more a "most" situation?

Reply:
I notice that in the current code we prefer "must":

```rust
        let yk = match self.yacckind {
            None => panic!("yacckind must be specified before processing."),
```

which feels clearer about what the user should do IMHO.

Reply:
Yeah, must is better!

Reply:
Should be fixed in [a307980](https://github.com/softdevteam/grmtools/pull/491/commits/a307980095753e5fa028e6de465eab90a7fd7e72)